params/precompile_config_test.go

```diff
@@ -7,19 +7,21 @@ import (
 	"math/big"
 	"testing"
 
+	"github.com/ava-labs/subnet-evm/commontype"
 	"github.com/ava-labs/subnet-evm/precompile"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestValidateWithChainConfig(t *testing.T) {
 	admins := []common.Address{{1}}
-	config := &ChainConfig{
-		PrecompileUpgrade: PrecompileUpgrade{
-			TxAllowListConfig: &precompile.TxAllowListConfig{
-				UpgradeableConfig: precompile.UpgradeableConfig{
-					BlockTimestamp: big.NewInt(2),
-				},
+	baseConfig := *SubnetEVMDefaultChainConfig
+	config := &baseConfig
+	config.PrecompileUpgrade = PrecompileUpgrade{
+		TxAllowListConfig: &precompile.TxAllowListConfig{
```

Comment:
nit: can we use `precompile.NewTxAllowListConfig` here for brevity?

Reply:
done